src/quo2/components/buttons/slide_button/view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo2.components.buttons.slide-button.view
+  (:require
+    [quo2.components.icon :as icon]
+    [quo2.foundations.colors :as colors]
+    [quo2.components.buttons.slide-button.style :as style]
+    [quo2.components.buttons.slide-button.utils :as utils]
+    [quo2.components.buttons.slide-button.animations :as animations]
+    [react-native.gesture :as gesture]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [oops.core :as oops]
+    [react-native.reanimated :as reanimated]))
+
+(defn- f-slider
+  [{:keys [disabled?]}]
+  (let [track-width        (reagent/atom nil)
+        sliding-complete?  (reagent/atom false)
+        gestures-disabled? (reagent/atom disabled?)
+        on-track-layout    (fn [evt]
+                             (let [width (oops/oget evt "nativeEvent" "layout" "width")]
```

Comment:
@flexsurfer, I see what he this function in the communities overview page and some others too. Should we put this in a common namespace, perhaps in the react-native folders or something like that. WDYT? 🤔

Reply:
i don't think it is worth it, but it could be a little bit shortened `(oops/oget evt "nativeEvent.layout.width")`